gen/definitions/classes.yaml

```
@@ -152,3 +152,9 @@ fvFBRMember:
   sub_category: "Networking"
   resource_notes:
     - "This resource should not be used in combination with the `vrf_fallback_route_group` nested attributes of other resources for the same object. Doing so will result in unexpected behaviour."
+
+l3extRsOutToFBRGroup:
+  resource_name: "l3out_fallback_route_group"
```

After discussion with Akini we should keep the standard here for any new relationship dedicated resources/datasources
```suggestion
  resource_name: "relation_to_fallback_route_group"
```